tmcprototype/SdpSubarrayLeafNode/SdpSubarrayLeafNode/SdpSubarrayLeafNode.py

```diff
@@ -542,7 +542,42 @@ def Scan(self, argin):
     @DebugIt()
     def EndScan(self):
         # PROTECTED REGION ID(SdpSubarrayLeafNode.EndScan) ENABLED START #
-        """ Ends scan"""
+        """
+        It invokes 'EndScan' command on SdpSubarray. This command is allowed when SdpSubarray is in SCANNING
+        state
```

Comment:
Add . at the end of sentence.

Reply:
Updated.
Commit Id: ac26cff01572307acfb8d209b0bd438358042414